images/linux/scripts/installers/python.sh

```diff
@@ -29,8 +29,8 @@ if isUbuntu18 || isUbuntu20 ; then
     python3 -m pipx ensurepath
 
     # Update /etc/environment
-    setEtcEnvironmentVariable "PIPX_BIN_DIR" $PIPX_BIN_DIR
-    setEtcEnvironmentVariable "PIPX_HOME" $PIPX_HOME
+    echo "PIPX_BIN_DIR=$PIPX_BIN_DIR" | tee -a /etc/environment
```

Comment:
Could you please describe why we shouldn't use `setEtcEnvironmentVariable` function?

addEtcEnvironmentVariable $variable_name $variable_value -> echo "$variable_name=\"$variable_value\"" | sudo tee -a /etc/environment

Reply:
We have `PIPX_BIN_DIR="/opt/pipx_bin"` in variables instead `PIPX_BIN_DIR=/opt/pipx_bin` and when we will try to install new packages pipx shows next message:
`Note: '/home/vsts/work/1/s/"/opt/pipx_bin"' is not on your PATH environment variable.`

New package will be installed to another place, not `/opt/pipx_bin`.

Reply:
Can we just remove quotes here: https://github.com/actions/virtual-environments/blob/446d7493c14b2aaf08befb32a50ae11dcc15ed92/images/linux/scripts/helpers/etc-environment.sh#L33 ?

Reply:
@maxim-lobanov , I believe it will be better to fix here:
`echo "$variable_name=\"$variable_value\"" | sudo tee -a /etc/environment`

Reply:
@vsafonkin  Yep, let's try

Reply:
Fixed

Reply:
cc: @dsame to review this PR

Reply:
@maxim-lobanov , i've just double checked, the quotes in `/etc/environment` does not play any important role - i've added them initially just for the convention - the original ubuntu `/etc/environment ` has the PATH variable quoted.

So the quotes should not leak to python installation IF `/etc/environment` read by PAM-aware shell. We did reboot in order to enforce PAM-aware `login` to re-read the environments. I guess the `reloadEtcEnvironment` used instead of reboot and that might cause the appearing of the quotes.

But my final verdict is "let's remove the quotes from `scripts/helpers/etc-environment.sh` "

Reply:
@dsame thank you for review!